[DCA-37] Move dns and cert creation out of cdk

ConsoleCatzirl opened the pull request:
Move the creation of the ACM certificate and DNS records out of CDK and into org-formation by referencing the ARN of an existing certificate and outputting the DNS name of the load balancer.

Depends on: https://github.com/Sage-Bionetworks-IT/organizations-infra/pull/731

ConsoleCatzirl committed:
[DCA-37] Move dns and cert creation out of cdk

Move the creation of the ACM certificate and DNS records out
of CDK and into org-formation by referencing the ARN of an existing
certifiate and outputting the DNS name of the load balancer.

zaro0508 commented on cdk.json:
```diff
@@ -36,9 +36,7 @@
-      "HOST_NAME": "dca.dnt-dev.sagebase.org",
-      "HOSTED_ZONE_NAME": "dnt-dev.sagebase.org",
-      "HOSTED_ZONE_ID": "Z0441843EAHKE7DR12AX",
+      "ACM_CERT_ARN": "arn:aws:acm:us-east-1:878654265857:certificate/a3b7c804-c1fc-4e58-bbc6-541ef2d65816",
```
this is a reference to `DCAProdAccount` however dev context is deployed to `DnTDevAccount`.  we will probably want to pass in a cert from DnTDevAccount.

Also can we add some documentation to describe how this works with what you added in Sage-Bionetworks-IT/organizations-infra?

jesusaurus replied:
whoops, thanks, apparently its very easy to copy+paste the wrong ARN

ConsoleCatzirl committed:
Use the dev cert in the dev context

The certificate needs to exist in the same account it will be referenced
in. Replace reference to dca-prod certifiate with dnt-dev certificate in
the "dev" context.

ConsoleCatzirl committed:
Describe integration with DNS and TLS in Readme
